server/controllers/Contacts.js

```diff
@@ -0,0 +1,67 @@
+const db = require("../models");
+const Contact = db.Contact;
+const SigninOption = db.SigninOption;
+const User = db.User;
+
+// Retrieve all Contact relationships in the database
+module.exports.getAllContacts = async (req, res) => {
+    let contacts = await Contact.findAll();
+    return res.send(contacts);
+}
+
+// Create and Save a new Contact
+module.exports.addContact = async (req, res) => {
+    let user_id = req.params.user_id;//temporarily getting the user_id from the url
+
+    // Validate request (need to add check to ensure user exists) + (prevent user from adding self as contact????)
+    if (
+        !req.body.contactEmail 
+        ) {
```

Comment:
This formatting is giving me ocd:

```suggestion
    if (!req.body.contactEmail) {
```